dcat/index.html

```diff
@@ -4635,6 +4635,13 @@ <h3>Compressed and packaged distributions</h3>
 <section id="changes" class="appendix">
     <h2>Change history</h2>
     <p>A full change-log is available on <a href="https://github.com/w3c/dxwg/commits/gh-pages/dcat">GitHub</a></p>
+    <section id="changes-since-20140116">
+        <h3>Changes since the W3C Recommendation of 4 February 2020</h3>
+        <p>The document has undergone the following changes since the W3C Recommendation of 4 February 2020 [[?VOCAB-DCAT-2-20200204]]:</p>
```

Comment:
@agbeltran this should add the version number. DCAT 1 is called DCAT 2014 in the spec. Do you want me to replace all the "DCAT 2014" with "DCAT 1" ?
```suggestion
        <p>The document has undergone the following changes since the DCAT 2 W3C Recommendation of 4 February 2020 [[?VOCAB-DCAT-2-20200204]]:</p>
```